astroquery/eso/core.py

```diff
@@ -67,6 +57,21 @@ def expired(self) -> bool:
         return time.time() > self.expiration_time - 600
 
 
+class QueryOnField():
+    table_name = ""
+    column_name = ""
+
+
+class QueryOnInstrument():
+    table_name = "dbo.raw"
+    column_name = "instrument"
+
+
+class QueryOnCollection():
+    table_name = "ivoa.ObsCore"
+    column_name = "obs_collection"
+
```

Comment:
I think it would be cleaner to have one dataclass storing table and column information and three constants with the different copnfigurations, example:

```
@dataclass
class QueryConfig:
  table_name: str
  column_name: str

QUERY_ON_FIELD = QueryConfig(table_name="", column_name="")
etc.
```

Reply:
Absolutely, thanks! This is what I was looking for but didn't manage to remember how it was done properly. 👍